core/models.py

```diff
@@ -41,3 +42,6 @@ class Review(models.Model):
     travel_allowance = models.IntegerField()
     pension = models.IntegerField()
     quality = models.IntegerField()
+
+    def get_absolute_url(self):
+        return reverse('review-detail', args=[str(self.pk)])
```

Comment:
I was following the django tutorial here: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/Generic_views